src/images/icons/Topics.js

```diff
@@ -1,42 +1,43 @@
-import * as React from "react";
+import * as React from 'react';
 
 function SvgTopics(props) {
```

Comment:
What is this image? Where can I see it?

Reply:
its icon of topic 
![image](https://user-images.githubusercontent.com/1830001/149099363-4be265d6-c19f-417a-96da-cd56ec2a65ab.png)

you can see on all buttons of create public bookmark, and on the homepage
![image](https://user-images.githubusercontent.com/1830001/149099872-897f9395-4747-4783-b7ca-508df83c43d6.png)